lib/modem_info/modem_info.c

```diff
@@ -362,6 +359,32 @@ static void flip_iccid_string(char *buf)
 	}
 }
 
+static void clip_response_status(char *buf)
```

Comment:
Somehow I'd like our libraries to be compatible with each other so this function should be handled in `at_cmd_parser` library (or `nrf_modem_at.h`) so that the user of `nrf_modem_at` can just pass its output to `at_cmd_parser`. That was the case with `at_cmd` and `at_cmd_parser`.

Reply:
I too think this should be handled by the parser, but if I made that change there now, it would not be compatible with libraries that still use `at_cmd` to receive the response. Once `at_cmd` is removed, we can make this change in the parser.

Reply:
Yep, I know there would be a compatibility issue but thought it would be possible to accept both versions. Might be easier to do as you said but we need to get that `at_cmd_parser` fixed for 1.9.0 to avoid use of current functionality in any new code.

Reply:
Alright, let me try and do this change in the parser and see how that looks.

Reply:
@trantanen see: #6386

Reply:
That PR looks good. I guess this PR should be marked with DNM label so that this function can be removed once 6386 is merged.

Reply:
Added `DNM`, until #6386 is in and this has been rebased.

Reply:
Rebased